R/utils.R

```diff
@@ -111,6 +111,27 @@ highlight_code = function(x) {
   paste(x, collapse = '\n')
 }
 
+highlight_output = function(x, options) {
+  highlight.output = options$highlight.output %||% FALSE
```

Comment:
I'd like to merge the two options `highlight.output` and `highlight.lines`. By `highlight.output = TRUE`, it means all output will be highlighted. By `highlight.output = indices`, highlight the specific lines.